### DISCUSSION TITLE
Frigate running in Docker Compose

### DISCUSSION THREAD

**Discussion by tlundste** [blakeblackshear/frigate#1496] [Ask A Question] [answered]
I do not have a Home Assistant Supervised installation.  I am running Home Assistant Core on Docker version, running Docker on an Intel NUC 8, i3 processor.
I see there are instructions for installing Frigate as part of Docker Compose installation.  I attempted that, but received the following error...
Pulling frigate (blakeblackshear/frigate:stable-amd)...
ERROR: manifest for blakeblackshear/frigate:stable-amd not found: manifest unknown: manifest unknown
I'm still pretty new to docker, so I could use a little help.
Is this the correct frigate install image to use?
Do I just need to add a repository where the image exists?  If so, what repository is that, and please help me with the command, and doing searches, I'm still having trouble finding that command.

Thank you.

**Comment by pjfian** [comment, marked as answer]
Have you missed the 64 off the end of the image name? 
The line in your docker-compose should be:
    image: blakeblackshear/frigate:stable-amd64

**Reply by tlundste** [reply]
Well, that's embarrassing.
Thanks!  I'll keep plodding on, and this time, I'll check my work more than once.
